content/guides/content/adding-papers.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Adding a new paper"
```

Comment:
Lets use the term "publication" here and throughout. "Adding a new publication to AnVIL".